vision/google/cloud/vision/face.py

```diff
@@ -22,6 +22,29 @@
 from google.cloud.vision.geometry import Position
 
 
+LIKELIHOOD_PB = {
+    0: 'UNKNOWN',
+    1: 'VERY_UNLIKELY',
+    2: 'UNLIKELY',
+    3: 'POSSIBLE',
+    4: 'LIKELY',
+    5: 'VERY_LIKELY',
+}
+
+
+def _get_pb_likelihood(likelihood):
+    """Convert gRPC Likelihood integer value to Likelihood instance.
+
+    :type likelihood: int
+    :param likelihood: Protobuf integer representing ``Likelihood``.
+
+    :rtype: :class:`~google.cloud.vision.likelihood.Likelihood`
+    :returns: Instance of ``Likelihood`` converted from gRPC value.
+    """
+    likelihood_pb = LIKELIHOOD_PB.get(likelihood)
```

Comment:
Where does `likelihood` come from the `pb2` module should have a way to get the enum